docs/includes/apiargs-MongoDBCollection-method-aggregate-option.yaml

```diff
@@ -35,6 +35,10 @@ post: |
   version of MongoDB, this option will be ignored.
 ---
 source:
+  file: apiargs-MongoDBCollection-method-find-option.yaml
+  ref: hint
+---
```

Comment:
Rather than embed the corresponding `find` option, let's write the body inline as is done in `apiargs-MongoDBCollection-method-count-options.yaml`. The main description paragraph is the same, but we'll want to add version information (related to [PHPLIB-284](https://jira.mongodb.org/browse/PHPLIB-284)) so users know which version of the library added support for this option:

```
arg_name: option
name: hint
type: string|array|object
description: |
  The index to use. Specify either the index name as a string or the index key
  pattern as a document. If specified, then the query system will only consider
  plans using the hinted index.

  .. versionadded:: 1.3
interface: phpmethod
operation: ~
optional: true
```